src/inc/plugins/ficons.php

```diff
@@ -147,7 +147,8 @@ function ficons_action(&$actions) {
 
 function ficons_show(&$forum) {
     global $templates, $forum_url, $mybb;
-    
+
+    $forum['icon'] = "";
```

Comment:
I guess the main issue was that `$forum['icon']` wasn't defined by default.

Reply:
Can you please redo it adding only this code? Because it has many changes because of spaces. Also it contains conflicts with new commits.
If you can't I'll do it tomorrow.
Thanks

Reply:
Sorry, I'm late .. as I see you have changed it already ... thx!